### ISSUE TITLE
Can we use neural network define by tensorflow as recurrent_fn in muzero?

### ISSUE THREAD

**Issue by hejujie** [google-deepmind/mctx#6]
We are using muzero , and define network and loss with tensorflow and mcts with naive python sentences；
But the mcts is very slow here, so we want to use mctx as a black box to replace our mcts, but use neural network in tensorflow.
We are not familiar with jax, the issue is whether we can just use a tensorflow `module` as parameter `recurrent_fn` in `muzero_policy`.

**Comment by fidlej** [comment]
You may try [tf2jax](https://github.com/deepmind/tf2jax) or [jax2tf.call_tf](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/README.md#calling-tensorflow-functions-from-jax).

**Comment by hejujie** [comment]
> You may try [tf2jax](https://github.com/deepmind/tf2jax) or [jax2tf.call_tf](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/README.md#calling-tensorflow-functions-from-jax).

Thanks!
